akka-distributed-data/src/main/scala/akka/cluster/ddata/ORMap.scala

```diff
@@ -216,7 +216,7 @@ final class ORMap[A, B <: ReplicatedData] private[akka] (
 
   /**
    * Adds an entry to the map
-   * @see [[#put]]
+   * @see [[ORMap#put(node:akka\.cluster\.ddata\.SelfUniqueAddress*]]
```

Comment:
wow, is that how to do it
what is the syntax? regular expression, but not really (escaping `.` and ending with `*`)

Reply:
Yeah it's weird. I followed https://stackoverflow.com/a/31569861/354132